vendor/revisions.json

```
@@ -1,4 +1,8 @@
 {
+  "v17": [
+    "17.0",
```

It probablt should be 
```suggestion
    "17rc1",
```